src/Clock.tsx

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable no-console */
```

Comment:
better not to disable for the whole document, there's a chance you gonna miss to remove some redundant logs. It is better to disable linter for the specific lines of your code